packages/jsx/README.md

```diff
@@ -233,13 +233,83 @@ export const HeaderInput = ({ size = 0 }) => {
 }
 ```
 
+It's recommended to wrap related elements in components because autogenerated class names include the component name, making them more readable and easier to identify:
+
+```html
+<input
+  class="HeaderInput_123456"
+  style="--size: 0;"
+/>
+```
+
 ### Components
 
-Components in `@reatom/jsx` are just functions returning JSX elements. They neither have state nor any lifecycle associated with them. Because component instantiation boils down into function calls, features like `$spread` are not supported in them.
+Components in `@reatom/jsx` are simple functions that return JSX elements. They have no state or lifecycle and don't support features like `$spread` since they are just function calls. This makes them lightweight and efficient during execution.
```

Comment:
```suggestion
Components in `@reatom/jsx` are simple functions that return native DOM elements. They have no state or lifecycle and don't support features like `$spread` since they are just function calls. This makes them lightweight and efficient during execution.
```